content/operate/guides/cosmos/syncing.md

```diff
@@ -4,6 +4,242 @@ title: Syncing
 
-_Coming Soon_
+---
+
+## Install firehose-cosmos
+
+`firehose-cosmos`, is an application that runs a few small, isolated processes,
+that together form the `Firehose` stack. A thorough discussion of the [Concepts & Architecture]({{< ref "/operate/concepts" >}})
+is discussed elsewhere. Needless to say, you must run `firehose-cosmos` to run a `Firehose` locally.
+
+You can download the latest version of `firehose-cosmos` [here](https://github.com/figment-networks/firehose-cosmos/releases),
+however we recommend you use the Dockerfile provided in the `firehose-cosmos` repository located [here](https://github.com/figment-networks/firehose-cosmos).
+
+As of the time of writing this, the latest release is v0.4.0, you can get a copy of this from [Figment's Docker Hub](https://hub.docker.com/r/figmentnetworks/firehose-cosmos/tags). The latest release will always be listed on the [Github Releases Page](https://github.com/figment-networks/firehose-cosmos/releases). Ensure you always use the most up to version of `firehose-cosmos` to ensure you have the latest functionality.
+
+```bash
+docker run --rm -it figmentnetworks/firehose-cosmos:0.4.0 /app/firehose help
+```
+
+Alternatively you can clone the repository and run `make install` to install it locally.
+
+To verify the installation was successful, run:
+
+```bash
+firehose-cosmos --version
+# or
+docker run --rm -it figmentnetworks/firehose-cosmos:0.4.0 /app/firehose --version
+```
+---
+
+## Install and run instrumented nodes
+
+In order to index Cosmos based nodes with `firehose-cosmos`, you will need to insure you are using the correctly modified binaries for the chain you are targetting. Figment provide these for CosmosHub and you can find them on the `Gaia-DM` releases page located [here](https://github.com/figment-networks/gaia-dm/releases). Alternatively, they also provide the prebuilt Docker Images which you can find listed on [their Dockerhub](https://hub.docker.com/r/figmentnetworks/firehose-cosmos/tags). These prebuilt Dockerfiles also start the `firehose-cosmos` process for you. Below is an example for v7.0.4 of the Osmosis testnet. You can modify this to use whichever version of the chain node you are indexing.
+
+```bash
+docker run --rm -it figmentnetworks/firehose-cosmos:fh-v0.4.0-osmosis1-testnet-v7.0.4 /app/firehose start
+```
+
+In the situation that you wish the run the nodes outside of Docker, you would need to ensure you are running the Figment modified node and then change the node configuration file to include the following:
+
+```ini
+#######################################################
+###       Extractor Configuration Options           ###
+#######################################################
+[extractor]
+enabled = true
+output_file = "stdout"
+```
+
+## Firehose-Cosmos Configuration
+
+If you wish to use a configuration file instead of setting all CLI flags, you can pass it with a `-c ./configfile.ini` argument.
```

Comment:
Is `configfile.ini` the right name? I thought it was a YAML file.

Reply:
I believe you can use either 🤔 i'll update to YAML to match the readme though